components/avatar.html

```diff
@@ -0,0 +1,200 @@
+              style="
+                width: 100%;
+                height: 201px;
+                margin-top: var(--spacing-md);
+                border: 2px solid rgb(221, 221, 221);
+                border-radius: 10px;
+                padding: var(--spacing-xl) var(--spacing-md);
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </section>
+          <section class="doc-preview-wrapper flex flex-col" id="squared">
+            <h2 class="doc-preview-heading">Squared</h2>
+            <p class="doc-preview-desc">
+              You can change the full style towards a squared
+              <span class="highlight-text">Avatar</span> with the
+              <span class="highlight-text">avatar-square</span> class.
+            </p>
+            <div
+              class="doc-preview-row doc-preview flex ai-center jc-start flex-wrap flex-gap"
+            >
+              <span class="avatar avatar-square">A</span>
+              <img
+                class="avatar avatar-square"
+                src="https://nextui.org/avatars/avatar-1.png"
+                alt="avatar"
+              />
+              <span class="avatar avatar-square">M</span>
+              <img
+                class="avatar avatar-square"
+                src="https://nextui.org/avatars/avatar-3.png"
+                alt="avatar"
+              />
+              <span class="avatar avatar-square">V</span>
+            </div>
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=one-light&wt=sharp&l=htmlmixed&width=678&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Fira+Code&fs=14px&lh=152%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522avatar%2520avatar-square%2522%253EA%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520class%253D%2522avatar%2520avatar-square%2522%2520src%253D%2522avatar-1.png%2522%2520%2520alt%253D%2522avatar%2522%252F%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522avatar%2520avatar-square%2522%253EM%253C%252Fspan%253E%250A%2520%2520%2520%2509%2520%2520%2520%253Cimg%2520class%253D%2522avatar%2520avatar-square%2522%2520src%253D%2522avatar-3.png%2522%2520%2520alt%253D%2522avatar%2522%252F%253E%250A%2520%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522avatar%2520avatar-square%2522%253E%2520A%253C%252Fspan%253E%250A%250A"
+              style="
```

Comment:
Everything looks good just a tip.

Create an account on carbon and then generate an iframe link you will get a shorter iframe source url.

Reply:
Thanks, Puneet, I did use the short link but there is some backend problem in carbon.sh so that's why I opt-out of this.